app/assets/stylesheets/style.scss

```diff
@@ -1,3 +1,31 @@
+body {
+  background-image: url("http://i.imgur.com/y6o43z3.jpg");
```

Comment:
So it's preferable not to use images for stuff like the header text on your page, since it can't be read by search engines or screen readers and is less adaptable to mobile. For now this is fine but if you guys would be up for it, I'd be happy to work with you sometime on re-creating the stuff in this image with pure HTML and CSS.